src/security-rules/security-rules.ts

```diff
@@ -115,6 +115,18 @@ export class SecurityRules implements FirebaseServiceInterface {
     return this.getRulesetForRelease(SecurityRules.CLOUD_FIRESTORE);
   }
 
+  /**
+   * Releases the specified ruleset as the current Cloud Firestore ruleset. This makes the specified ruleset
+   * the currently applied ruleset for Cloud Firestore.
```

Comment:
Total nit; seems like the second sentence is more complete and could stand on its own here.

Reply:
Done